docs/standard/native-interop/best-practices.md

```diff
@@ -126,6 +128,10 @@ public struct UnicodeCharStruct
 
 You can see if a type is blittable or contains blittable contents by attempting to create a pinned `GCHandle`. If the type isn't a string or considered blittable, `GCHandle.Alloc` will throw an `ArgumentException`.
 
+### Blittable types when runtime marshalling is disabled
+
+When [runtime marshalling is disabled](disabled-marshalling.md), the rules for which types are blittable are significantly simpler. All types that are [C# `unmanaged`](../../csharp/language-reference/builtin-types/unmanaged-types.md) types are blittable. All types that are not C# `unmanaged` types are not blittable. The concept of "types with blittable contents" does not apply when runtime marshalling is disabled.
```

Comment:
Would it be worth clarifying that this largely impacts `bool` and `char` on RyuJIT (.NET 6+). Other implementations may of course have their own differences here, but calling out the "common" case seems helpful at least.